Add support for KNX DPT 14.058

2015-10-05 15:14:50.490 [ERROR] [i.internal.GenericItemProvider] - Binding configuration of type 'knx' of item ‘WS_Luftdrucksensor_Pa‘ could not be parsed correctly.
org.openhab.model.item.binding.BindingConfigParseException: DPT 14.058 is not supported by the KNX binding.

DPT 14.058 is used for displaying air pressure. This value is needed for calculating the local weather forecast, especially for the forecast of rain and hail.

--- a/bundles/binding/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/dpt/KNXCoreTypeMapper.java
+++ b/bundles/binding/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/dpt/KNXCoreTypeMapper.java
@@ -190,6 +190,7 @@ public class KNXCoreTypeMapper implements KNXTypeMapper {
         dptTypeMap.put(DPTXlator4ByteFloat.DPT_ELECTRIC_POTENTIAL.getID(), DecimalType.class);
         dptTypeMap.put(DPTXlator4ByteFloat.DPT_FREQUENCY.getID(), DecimalType.class);
         dptTypeMap.put(DPTXlator4ByteFloat.DPT_POWER.getID(), DecimalType.class);
+        dptTypeMap.put(DPTXlator4ByteFloat.DPT_PRESSURE.getID(), DecimalType.class);
 
         // Datapoint Types "String", Main number 16
         dptTypeMap.put(DPTXlatorString.DPT_STRING_8859_1.getID(), StringType.class);

